Src/Fido2NetLib.csproj

```diff
@@ -23,17 +22,19 @@
     <PackageProjectUrl>https://github.com/abergs/fido2-net-lib</PackageProjectUrl>
     <PackageLicenseUrl>https://github.com/abergs/fido2-net-lib/blob/master/LICENSE.txt</PackageLicenseUrl>
   </PropertyGroup>
-
   <ItemGroup>
+    <PackageReference Include="Fido2.Models" Version="1.0.0-preview5" />
```

Comment:
If we were to release both packages as one deployment object but two artifacts this packagereference would not be required.

Reply:
I'm intrigued, could you explain more?

Reply:
With csprojs doing "dotnet pack" on a csproj will automatically pack all projects referenced as well while adding nuspec dependencies to them simultaneously (unless specifically disabled of course).
With a shared Version variable this means that dotnet pack on FIDO2 will pack both Fido2.nupkg and Fido2.Models.nupkg with the versions and dependencies automagically set.

Reply:
Sounds great!